packages/frontend/src/pages/Users/components/UsersTablePagination.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+  faAngleLeft,
+  faAngleRight,
+  faAnglesLeft,
+  faAnglesRight,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+interface IUsersTablePagination {
+  lastPage: number;
+  page: number;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+}
+
+const UsersTablePagination = ({
+  lastPage,
+  page,
+  setPage,
+}: IUsersTablePagination): JSX.Element => {
+  return (
+    <div className="flex gap-2 items-center select-none mt-8 pl-4">
```

Comment:
Let's increase the spacing between these arrow elements slightly to reflect the mockup:

![image](https://user-images.githubusercontent.com/159270/168446967-5900c45b-8527-4072-902a-f0b274468291.png)